module/zcommon/zfeature_common.c

```diff
@@ -731,6 +731,12 @@ zpool_feature_init(void)
 	    ZFEATURE_FLAG_READONLY_COMPAT, ZFEATURE_TYPE_BOOLEAN, NULL,
 	    sfeatures);
 
+	zfeature_register(SPA_FEATURE_AVZ_V2,
+	    "com.klarasystems:avz_v2", "avz_v2",
```

Comment:
How about "vdev_zaps_v2" instead.  I suspect "avz_v2" will only make sense to those already very familiar with the code.